services/core-web/src/tests/components/modalContent/UpdateMinespaceUserModal.spec.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { shallow } from "enzyme";
+import UpdateMinespaceUserModal from "@/components/modalContent/UpdateMinespaceUserModal";
+
```

Comment:
This should be using the function/class import instead of the default. This component isn't connected to redux, so the test is passing, but if it was a connected component the test would fail. It's a good habit to only use the class/functional import when testing components. 

```import { UpdateMinespaceUserModal } from "@/components/modalContent/UpdateMinespaceUserModal"```